<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="main" stretch="1,12">
     <item>
      <layout class="QHBoxLayout" name="top" stretch="2,2,5">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="back">
         <property name="text">
          <string>BACK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="title">
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="settings_layout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>821</width>
            <height>482</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="content" stretch="0,0,0,0,0,0,0,0,0,1">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>40</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>40</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QGroupBox" name="appearance_group">
               <property name="title">
                <string>Appearance</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>15</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>15</number>
                </property>
                <item>
                 <layout class="QFormLayout" name="theme_layout">
                  <property name="horizontalSpacing">
                   <number>20</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>10</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="theme_label">
                    <property name="text">
                     <string>Theme:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="theme_selection">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="font_size_layout">
                  <property name="horizontalSpacing">
                   <number>20</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>10</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="font_size_label">
                    <property name="text">
                     <string>Font Size:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="font_size_spinbox">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="suffix">
                     <string> pt</string>
                    </property>
                    <property name="minimum">
                     <number>8</number>
                    </property>
                    <property name="maximum">
                     <number>24</number>
                    </property>
                    <property name="value">
                     <number>14</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="regional_group">
               <property name="title">
                <string>Regional Settings</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>15</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>15</number>
                </property>
                <item>
                 <layout class="QFormLayout" name="language_layout">
                  <property name="horizontalSpacing">
                   <number>20</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>10</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="language_label">
                    <property name="text">
                     <string>Language:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="language_selection">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="date_format_layout">
                  <property name="horizontalSpacing">
                   <number>20</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>10</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="date_format_label">
                    <property name="text">
                     <string>Date Format:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="date_format_selection">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="time_format_layout">
                  <property name="horizontalSpacing">
                   <number>20</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>10</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="time_format_label">
                    <property name="text">
                     <string>Time Format:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="time_format_selection">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="notifications_group">
               <property name="title">
                <string>Notifications</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>15</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>15</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="enable_notifications">
                  <property name="text">
                   <string>Enable desktop notifications</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="enable_sound">
                  <property name="text">
                   <string>Enable notification sounds</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="data_group">
               <property name="title">
                <string>Data Management</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>15</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>15</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="auto_backup">
                  <property name="text">
                   <string>Enable automatic database backup</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="backup_buttons_layout">
                  <property name="spacing">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="backup_now_btn">
                    <property name="text">
                     <string>Backup Now</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="restore_backup_btn">
                    <property name="text">
                     <string>Restore Backup</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="backup_spacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="academic_group">
               <property name="title">
                <string>Academic Defaults</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>15</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>15</number>
                </property>
                <item>
                 <layout class="QFormLayout" name="default_semester_layout">
                  <property name="horizontalSpacing">
                   <number>20</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>10</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="default_semester_label">
                    <property name="text">
                     <string>Default Semester:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="default_semester_selection">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="academic_year_layout">
                  <property name="horizontalSpacing">
                   <number>20</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>10</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="academic_year_label">
                    <property name="text">
                     <string>Academic Year:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="academic_year_selection">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="advanced_group">
               <property name="title">
                <string>Advanced</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>15</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>15</number>
                </property>
                <item>
                 <layout class="QFormLayout" name="rows_per_page_layout">
                  <property name="horizontalSpacing">
                   <number>20</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>10</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="rows_per_page_label">
                    <property name="text">
                     <string>Table Rows Per Page:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="rows_per_page_spinbox">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="minimum">
                     <number>10</number>
                    </property>
                    <property name="maximum">
                     <number>100</number>
                    </property>
                    <property name="singleStep">
                     <number>10</number>
                    </property>
                    <property name="value">
                     <number>50</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="confirm_delete">
                  <property name="text">
                   <string>Confirm before deleting records</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="reset_layout">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <spacer name="reset_spacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="reset_defaults_btn">
                 <property name="text">
                  <string>Reset to Defaults</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="status_label">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="bottom_spacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
